pkg/operator/operator.go

```diff
@@ -222,7 +222,9 @@ func NewOperator() (context.Context, *Operator) {
 	lo.Must0(mgr.GetFieldIndexer().IndexField(ctx, &storagev1.VolumeAttachment{}, "spec.nodeName", func(o client.Object) []string {
 		return []string{o.(*storagev1.VolumeAttachment).Spec.NodeName}
 	}), "failed to setup volumeattachment indexer")
-
+	lo.Must0(mgr.AddReadyzCheck("manager", func(req *http.Request) error {
```

Comment:
Doesn't this require us to drop the webhook first? Should probably do that before adding this check back.

Reply:
Yep right, working on that change.